spring-xd-module/src/test/java/org/springframework/xd/module/core/ModuleFactoryTests.java

```diff
@@ -58,6 +58,25 @@ public void createXmlModule() {
 		createResourceConfiguredModule("xmlModule",ModuleType.processor);
 	}
 
+	@Test(expected = IllegalStateException.class)
+	public void multipleXmlConfigThrowsException() {
+		createResourceConfiguredModule("invalid",ModuleType.processor);
+	}
+
+	@Test(expected = IllegalArgumentException.class)
+	public void missingConfigThrowsException() {
+		ModuleDefinition moduleDefinition = ModuleDefinitions.simple("missing", ModuleType.processor,
+				"classpath:/ModuleFactoryTests/modules/processor/missing");
+		ModuleDescriptor moduleDescriptor = new ModuleDescriptor.Builder()
```

Comment:
Just browsing through the web interface at the moment, but I don't see `moduleDescriptor` being used in the test